go/oasis-node/cmd/genesis/genesis.go

```diff
@@ -157,6 +157,7 @@ func doInitGenesis(cmd *cobra.Command, args []string) {
 		ChainID:   chainID,
 		Time:      time.Now(),
 		HaltEpoch: epochtime.EpochTime(viper.GetUint64(cfgHaltEpoch)),
+		Height:    1,
```

Comment:
Note that this height is currently unused as the underlying consensus backend (Tendermint) does not yet support starting at heights different from 1.

Reply:
It's exported in the genesis document and the `GetStatus` method, which is used by external services (e.g. our Rosetta gateway) to determine the height of the genesis block, so having a mismatching value here breaks other things :)